src/test/msgr/test_msgr.cc

```diff
@@ -722,6 +722,25 @@ TEST_P(MessengerTest, AuthTest) {
   ASSERT_TRUE(conn->is_connected());
   ASSERT_EQ(1U, static_cast<Session*>(conn->get_priv().get())->get_count());
 
+  // 3. krb / mix auth 
```

Comment:
will this work in an environment without a configured kerberos? I'd assume it wouldn't. As such, running it as part of the messenger test seems like a great way to break both our CI and tests for everyone.

Reply:
Working on Teuthology tests for this.